tests/cache/models/samples/get.js

```diff
@@ -76,6 +77,73 @@ describe('tests/cache/models/samples/get.js, ' +
     });
   });
 
+  it('get with wildcard and with cacheGetSamplesWildcard flag off ' +
+    'shold not cache response', (done) => {
+    tu.toggleOverride('cacheGetSamplesWildcard', false);
+    api.get(`${path}?name=___Subj*`)
+    .set('Authorization', token)
+    .expect(constants.httpStatus.OK)
+    .end((err, res) => {
+      if (err) {
+        return done(err);
+      }
+
+      redisCache.get('___Subj*', (cacheErr, reply) => {
```

Comment:
Any reason behind accessing Redis directly here, instead of using the api?

Reply:
Yes so that I can verify that it is cached.